### DISCUSSION TITLE
Should Data Quality Spec become an "official" FD spec?

### DISCUSSION THREAD

**Discussion by pwalsh** [frictionlessdata/frictionlessdata.io#861] [General]
https://github.com/frictionlessdata/data-quality-spec

Used in GoodTables.

**Comment by roll** [comment]
DQS has a little bit different nature than specs we have now. But for Frictionless Data project in overall I suppose it could be pretty logical move.

**Comment by rufuspollock** [comment]
I'd suggest we start with a pattern - let's get a PR for that in and go from there.

**Comment by Stephen-Gates** [comment]
See also https://github.com/frictionlessdata/tableschema-js/issues/77 
Where I suggest using DQS for consistent error messages.

**Comment by rufuspollock** [comment]
@pwalsh @roll could we get a mini-DQS pattern started. It would really help implementors as well I think.

**Comment by pwalsh** [comment]
@rufuspollock yes, I've assigned to myself to action this.
